addons/l10n_it_edi/models/account_move_send.py

```diff
@@ -30,6 +30,21 @@ def _get_alerts(self, moves, moves_data):
         if it_moves := moves.filtered(lambda m: 'it_edi_send' in moves_data[m]['extra_edis'] or moves_data[m]['invoice_edi_format'] == 'it_edi_xml'):
             if it_alerts := it_moves._l10n_it_edi_export_data_check():
                 alerts.update(**it_alerts)
+
+            # Invite the user to authorize Odoo and start using IT EDI in production mode
+            if 'prod' not in it_moves.company_id.l10n_it_edi_proxy_user_id.mapped('edi_mode'):
```

Comment:
Suppose we're selecting moves from 2 companies the user controls, shouldn't this be like...
```suggestion
            if 'prod' not in it_moves.mapped('company_id.l10n_it_edi_proxy_user_id.edi_mode'):
```
or something? I don't remember exactly.

Reply:
I think both will do the same thing, but yours makes more sense (one move will only have one [company-proxy_user-edi_mode]).
I changed it to `it_moves.mapped('l10n_it_edi_proxy_mode')` since we're already defining the field anyway :ok_hand: